.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
```

Comment:
how does this work and how does it help us ?

Reply:
https://docs.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically
https://dependabot.com/#how-it-works

Reply:
any github actions, docker base versions, golang deps etc., are automatically checked for security/regular updates and a PR is opened.